### PR TITLE
Bugfix/5.26.0 qa

### LINKED ISSUE #1736: QA Freighter v5.26.0-beta.0
Release + Context: https://stellarfoundation.slack.com/archives/C03347FNAHK/p1734023353188429

### PR DESCRIPTION
**Pull request by piyalbasu**
Closes #1736 

Addresses comments from Charles.

Many of these fixes are simple CSS fixes that are holdovers from upgrading SDS v2 to v3

### CODE REVIEW

File: extension/src/popup/Router.tsx [hunk 1]
```diff
@@ -277,7 +277,7 @@ const Outlet = () => {
     ((location.pathname === ROUTES.welcome &&
       applicationState === APPLICATION_STATE.MNEMONIC_PHRASE_CONFIRMED) ||
       SHOW_NAV_ROUTES.some((route) => location.pathname === route) ||
-      isSwap);
+      (isSwap && location.pathname !== ROUTES.unlockAccount));
```
Comment (piyalbasu) [inline extension/src/popup/Router.tsx:280]: ugh - sorry for adding even more twisty logic to this conditional: but we're basically saying that if we show the password auth screen during the Swap flow (bc the user's session happened to time out), don't show the bottom nav

File: extension/src/popup/components/EnterPassword/index.tsx [hunk 2]
```diff
@@ -39,7 +39,7 @@ export const EnterPassword = ({
   const titleLabel = title || t("Enter your password");
   const descriptionLabel = `${
     description || t("Enter your account password to verify your account.")
-  } ${t("You won’t be asked to do this again for the next 24 hours.")}`;
+  }`;
```
Comment (piyalbasu) [inline extension/src/popup/components/EnterPassword/index.tsx:42]: @CassioMG Heads up that after some more discussion, we decided to remove this line during QA

File: extension/src/popup/components/manageAssets/AddAsset/index.tsx [hunk 3]
```diff
@@ -151,8 +151,8 @@ export const AddAsset = () => {
           setVerifiedLists(verifiedTokens[0].verifiedLists);
           setAssetRows(
             verifiedTokens.map((record: VerifiedTokenRecord) => ({
-              code: record.code,
-              issuer: record.issuer,
+              code: record.code || record.contract,
+              issuer: record.issuer || record.contract,
```
Comment (piyalbasu) [inline extension/src/popup/components/manageAssets/AddAsset/index.tsx:155]: there is an edge case where a user's asset list can be missing fields but still pass the schema validation Orbit Lens created. In that case, we're adding some fallback values here

Comment (piyalbasu) [reply extension/src/popup/components/manageAssets/AddAsset/index.tsx:155]: I'll take to OL about tightening the schema rules

File: extension/src/popup/styles/global.scss [hunk 4]
```diff
@@ -113,8 +113,3 @@ a {
 .sds-theme-dark ol {
   color: inherit !important;
 }
-
-// SDS overrides
-.Heading--lg {
-  line-height: normal;
```
Comment (piyalbasu) [inline extension/src/popup/styles/global.scss:119]: I double checked that this override was only being used in one place so removing this global override should not have any unexpected results

File: extension/src/popup/Router.tsx (hunk 1 above)
Comment (aristidesstaffieri) [reply extension/src/popup/Router.tsx:280]: makes sense!

**Review by aristidesstaffieri — APPROVED** [review]

File: extension/src/popup/components/EnterPassword/index.tsx (hunk 2 above)
Comment (CassioMG) [reply extension/src/popup/components/EnterPassword/index.tsx:42]: fine by me!

**Pull request merged**